api/lib/database/helpers.js

```diff
@@ -59,15 +59,17 @@ function getColumns(objects) {
   return Object.keys(obj)
 }
 
-const insert = ({ hasura_url, hasura_admin_key }, upsert = false) => (
-  schema
-) => async (table, rows, height, chainId, returnKeys) => {
+const insert = (
+  { hasura_url, hasura_admin_key },
+  upsert = false,
+  usePrefix = true
```

Comment:
remove. Just don't send a schema and you get the root schema

Reply:
oh, good to know!